THEMING.md

```diff
@@ -90,7 +90,9 @@ As mentioned the theme is mostly based on `slate` but it has been further refine
 
 # Linting
 
-[Stylelint](https://stylelint.io/) has been setup to check for common errors on the css/less files. Note it is not run against the SUI files
+[Stylelint](https://stylelint.io/) has been setup to check for common errors on the css/less/sass file
+
+s. Note it is not run against the SUI files
 
 A tricky part of using `stylelint` is the way variables are checked.
 A small sed script can take the variables from either the dark or light theme and export them as a `vars.css` file.
```

Comment:
Is this section true anymore? Hadn't you removed it earlier?

Reply:
Ah yes, but when I attempted to make linting work again I undeleted it. I will edit.